src/components/BioButton/BioButton.ts

```diff
@@ -17,7 +17,9 @@ class BioButton extends Component<BioButtonProps, BioButtonState> {
 	registerEventListeners() {
 		this.addEventListener('click', (e: Event) => {
 			e.preventDefault();
-			if (this.props.url) {
+			if (this.props.url[0] === '#') {
+				window.scrollTo(0, document.querySelector(this.props.url).getBoundingClientRect().top);
```

Comment:
please check first of the element really exists, before getting porperties like its offset.
and: why didn you use .getBoundingClientRect().top instead of .offsetTop?